### ISSUE TITLE
Parameter doc

### ISSUE THREAD

**Issue by JuliusSustarevas** [pooyanjamshidi/brass_gazebo_battery#1]
Hi
Could you add a doc on what the parameters are? What units they are in  for example, its not always clear. The references you provided help but the first one doesn't load.  Would appreciate the help, thanks.

**Comment by pooyanjamshidi** [comment]
@JuliusSustarevas Please refer to [this](https://github.com/pooyanjamshidi/brass_gazebo_battery#acknowledgements).

**Comment by JuliusSustarevas** [comment]
Yes, the link 1 is not loading tho it seems to be just Ohms law,  the rest do not go through how you modelled the battery. Specifically what are the smooth_current_tau, linear_coef and constant_coef. Thanks for your help int he matter.

**Comment by mgerhardt10** [comment]
Duplicate of #1 

Might be something similar to: http://osrf-distributions.s3.amazonaws.com/gazebo/api/7.0.0/classgazebo_1_1LinearBatteryPlugin.html

**Comment by mgerhardt10** [comment]
@JuliusSustarevas

Reading through the code, here's what I've come up with for discharging:

![image](http://www.sciweavers.org/upload/Tex2Img_1591890734/render.png)
![image](http://www.sciweavers.org/upload/Tex2Img_1591890864/render.png)

**Note**: dt is a Gazebo param and defaults to 0.001

![image](http://www.sciweavers.org/upload/Tex2Img_1591891002/render.png)
**Initial Values**:
q = q0 = initialCharge

I hope I didn't miss anything

**Comment by pooyanjamshidi** [comment]
@mgerhardt10 I cannot see the images you put in there, could you please check them out?

**Comment by mgerhardt10** [comment]
> @mgerhardt10 I cannot see the images you put in there, could you please check them out?

I'm able to see them if I click the image link. For reference, they're just the equations from your discharge_battery.cpp file
